src/ScheduledExportBundle/Controller/GridConfigController.php

```diff
@@ -25,18 +25,26 @@ class GridConfigController extends AdminController
      */
     public function getListAction(Request $request)
     {
+        $user = $this->getAdminUser();
         $gridConfigs = new GridConfig\Listing();
+        $gridConfigs->setCondition("ownerId = ? or shareGlobally = 1", [$user->getId()]);
```

Comment:
What is "1" ? maybe you should use const variable for that?

Reply:
It's how Pimcore saves true in db.